### PR TITLE
IP-3232: Replicate dynamic MoE for all other countries

### PR DESCRIPTION
**Pull request by macnguyen**

### CODE REVIEW

File: hk/mapofecommerce/data/translation.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   "contentMerchants": "E&#8209;commerce who received more than 100,000 visitors per-month or 100,000 social media followers are qualified for this list. <br />The following industries were not included in this list: e-ticketing, financial services, rental services, insurance, delivery service, food & beverage, meta-search, couponing and cashback websites. All mentioned in this list was founded by individuals and was initiated in the e-commerce sector.",
+   "moreDetailData": "Click here to view data in detail: ",
+   "moreDataLink": "",
+   "seeUpdateData": "To update data: ",
+   "updateDataLink": "",
+   "IEMAAwards": "IEMA Awards<Br />iPrice E&#8209;Commerce Merchants Award (iEMA) 2016 is South East Asia preeminent awards voted and judged by online consumers. For more information",
+   "embedPageTitle": "Embed this infographic on your site",
+   "countryName": "Hong Kong"
+ }
+]
```
Comment (danielbk08) [inline hk/mapofecommerce/data/translation.json]: EOL

**Comment by MohdFarith** [comment]
LGTM for QA.

File: my/mapofecommerce/data/translation.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   "contentMerchants": "E-commerce who received more than 100,000 visitors per-month or 100,000 social media followers are qualified for this list. <br />The following industries were not included in this list: e-ticketing, financial services, rental services, insurance, delivery service, food & beverage, meta-search, couponing and cashback websites. All mentioned in this list was founded by individuals and was initiated in the e-commerce sector.",
+   "moreDetailData": "Click here to view data in detail: ",
+   "moreDataLink": "http://bit.ly/datamalaysia",
+   "seeUpdateData": "Something does not look right? Inform us through this form: ",
+   "updateDataLink": "https://docs.google.com/forms/d/e/1FAIpQLSeZZHU-HcKYHlZ1lnJ7bIUTkNalRFcxl1hpUpSTOu8J3YCi8w/viewform?usp=sf_link",
+   "IEMAAwards": "IEMA Awards<br />iPrice E-Commerce Merchants Award (iEMA) 2016 is South East Asia preeminent awards voted and judged by online consumers. Learn more here",
+   "embedPageTitle": "Embed this infographic on your site",
+   "countryName": "Malaysia"
+ }
+]
```
Comment (danielbk08) [inline my/mapofecommerce/data/translation.json]: EOL

File: ph/mapofecommerce/data/translation.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+   "contentMerchants": "E&#8209;commerce who received more than 100,000 visitors per-month or 100,000 social media followers are qualified for this list. <br />The following industries were not included in this list: e-ticketing, financial services, rental services, insurance, delivery service, food & beverage, meta-search, couponing and cashback websites. All mentioned in this list was founded by individuals and was initiated in the e-commerce sector.",
+   "moreDetailData": "Click here to view data in detail: ",
+   "moreDataLink": "",
+   "seeUpdateData": "To update data: ",
+   "updateDataLink": "",
+   "IEMAAwards": "IEMA Awards<Br />iPrice E&#8209;Commerce Merchants Award (iEMA) 2016 is South East Asia preeminent awards voted and judged by online consumers. For more information",
+   "embedPageTitle": "Embed this infographic on your site",
+   "countryName": "Philippines"
+ }
+]
```
Comment (danielbk08) [inline ph/mapofecommerce/data/translation.json]: EOL

File: sg/mapofecommerce/data/translation.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+   "contentMerchants": "E&#8209;commerce who received more than 100,000 visitors per-month or 100,000 social media followers are qualified for this list. <Br />The following industries were not included in this list: e-ticketing, financial services, rental services, insurance, delivery service, food & beverage, meta-search, couponing and cashback websites. All mentioned in this list was founded by individuals and was initiated in the e-commerce sector.",
+   "moreDetailData": "Click here to view data in detail: ",
+   "moreDataLink": "http://bit.ly/datasingapore",
+   "seeUpdateData": "To update data: ",
+   "updateDataLink": "https://docs.google.com/forms/d/e/1FAIpQLSdmWGmddDzpje_zrVkkmZ99svMP4NDn3FKjwotUufmQ4spIMg/viewform?usp=sf_link",
+   "IEMAAwards": "IEMA Awards<Br />iPrice E&#8209;Commerce Merchants Award (iEMA) 2016 is South East Asia preeminent awards voted and judged by online consumers. For more information",
+   "embedPageTitle": "Embed this infographic on your site",
+   "countryName": "Singapore"
+ }
+]
```
Comment (danielbk08) [inline sg/mapofecommerce/data/translation.json]: EOL

File: th/mapofecommerce/data/translation.json [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+   "contentMerchants": "E&#8209;commerce who qualified for this list meets the minimum requirement of 100,000 visitors per-month or 100,000 social media followers. <br />The following industries were not included in this list: e-ticketing, financial services, rental services, insurance, delivery service, food & beverage, meta-search, couponing and cashback websites. All mentioned in this list was founded by individuals and was initiated in the e-commerce sector.",
+   "moreDetailData": "Click here to view data in detail:",
+   "moreDataLink": "http://bit.ly/datathailand",
+   "seeUpdateData": "To update the data :",
+   "updateDataLink": "https://docs.google.com/forms/d/e/1FAIpQLSdGFtEBd0Pxkh7Fcgv2PXYJSvMGhQR68x7AcZGNkBn1qijONg/viewform?usp=sf_link",
+   "IEMAAwards": "IEMA Awards<br />iPrice E&#8209;Commerce Merchants Award (iEMA) 2016 is South East Asia preeminent awards voted and judged by online consumers. For more information",
+   "embedPageTitle": "Embed this infographic on your site",
+   "countryName": "Thailand"
+ }
+]
```
Comment (danielbk08) [inline th/mapofecommerce/data/translation.json]: EOL

File: vn/mapofecommerce/data/translation.json [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+   "contentMerchants": "E-commerce who received more than 100,000 visitors per-month or 100,000 social media followers are qualified for this list. <br />The following industries were not included in this list: e-ticketing, financial services, rental services, insurance, delivery service, food & beverage, meta-search, couponing and cashback websites. All mentioned in this list was founded by individuals and was initiated in the e-commerce sector.",
+   "moreDetailData": "Click here to view data in detail: ",
+   "moreDataLink": "http://bit.ly/datavietnam",
+   "seeUpdateData": "To update the data",
+   "updateDataLink": "https://docs.google.com/forms/d/e/1FAIpQLScb6wfQF874yF7mGSknmKeigvX1hiVCh8rdyVoqnUp4Dt4LqQ/viewform?usp=sf_link",
+   "IEMAAwards": "IEMA Awards<br />iPrice E-Commerce Merchants Award (iEMA) 2016 is South East Asia preeminent awards voted and judged by online consumers. For more information",
+   "embedPageTitle": "Embed this infographic on your site",
+   "countryName": "Vietnam"
+ }
+]
```
Comment (danielbk08) [inline vn/mapofecommerce/data/translation.json]: EOL

**Pull request merged**